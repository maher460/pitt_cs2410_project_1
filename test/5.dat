ori R20, R0, 100
ori R21, R0, 1
ori R22, R0, 0
ori R23, R0, 1
fld F1, 116(R0)
fld F2, 120(R0)
fld F3, 124(R0)
fld F4, 128(R0)
fld F5, 132(R0)
L4:add R22, R22, R23
fadd F10, F1, F2
fsub F11, F10, F3
fmul F12, F11, F4
sub R20, R20, R21
fmul F13, F2, F3
fdiv F14, F4, F2
addi R23, R23, 1
beq R20, $0, L5
bne R20, $0, L4
L5:fsd F10, 136(R0)
fsd F11, 140(R0)
fsd F12, 144(R0)
fsd F13, 148(R0)
fsd F14, 152(R0)
sd R22, 196(R0)
DATA
Mem(116) = 1.0
Mem(120) = 2.0
Mem(124) = 3.0
Mem(128) = 4.0
Mem(132) = 5.0
Mem(136) = 1.0
Mem(140) = 2.0
Mem(144) = 3.0
Mem(148) = 4.0
Mem(152) = 5.0
Mem(196) = 0.0