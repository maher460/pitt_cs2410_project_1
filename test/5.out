----------
Data Cache
----------
Mem(116) = 1.0
Mem(120) = 2.0
Mem(124) = 3.0
Mem(128) = 4.0
Mem(132) = 5.0
Mem(136) = 3.0
Mem(140) = 0.0
Mem(144) = 0.0
Mem(148) = 6.0
Mem(152) = 2.0
Mem(196) = 5050.0
-----------------
Integer Registers
-----------------
Int_Reg[0]=0
Int_Reg[1]=0
Int_Reg[2]=0
Int_Reg[3]=0
Int_Reg[4]=0
Int_Reg[5]=0
Int_Reg[6]=0
Int_Reg[7]=0
Int_Reg[8]=0
Int_Reg[9]=0
Int_Reg[10]=0
Int_Reg[11]=0
Int_Reg[12]=0
Int_Reg[13]=0
Int_Reg[14]=0
Int_Reg[15]=0
Int_Reg[16]=0
Int_Reg[17]=0
Int_Reg[18]=0
Int_Reg[19]=0
Int_Reg[20]=0
Int_Reg[21]=1
Int_Reg[22]=5050
Int_Reg[23]=101
Int_Reg[24]=0
Int_Reg[25]=0
Int_Reg[26]=0
Int_Reg[27]=0
Int_Reg[28]=0
Int_Reg[29]=0
Int_Reg[30]=0
Int_Reg[31]=0
------------------------
Floating Point Registers
------------------------
FP_Reg[0]=0.0
FP_Reg[1]=1.0
FP_Reg[2]=2.0
FP_Reg[3]=3.0
FP_Reg[4]=4.0
FP_Reg[5]=5.0
FP_Reg[6]=0.0
FP_Reg[7]=0.0
FP_Reg[8]=0.0
FP_Reg[9]=0.0
FP_Reg[10]=3.0
FP_Reg[11]=0.0
FP_Reg[12]=0.0
FP_Reg[13]=6.0
FP_Reg[14]=2.0
FP_Reg[15]=0.0
FP_Reg[16]=0.0
FP_Reg[17]=0.0
FP_Reg[18]=0.0
FP_Reg[19]=0.0
FP_Reg[20]=0.0
FP_Reg[21]=0.0
FP_Reg[22]=0.0
FP_Reg[23]=0.0
FP_Reg[24]=0.0
FP_Reg[25]=0.0
FP_Reg[26]=0.0
FP_Reg[27]=0.0
FP_Reg[28]=0.0
FP_Reg[29]=0.0
FP_Reg[30]=0.0
FP_Reg[31]=0.0

commit instructions: 1014